DESCRIPTION

```diff
@@ -43,7 +43,8 @@ Imports:
     rmarkdown,
     stringr,
     tibble,
-    tidyselect
+    tidyselect,
+    tidyr
```

Comment:
When bbf34c2 (bug fix: fix display of multiple test directories, 2023-09-18) introduced the use of `tidyr` in an `R/` file, it should have moved `tidyr` from `Suggests` to `Imports`.

```
$ git grep tidyr bbf34c2^ -- R
# empty
$ git grep tidyr bbf34c2 -- R
bbf34c2:R/format-report.R:      ) %>% tidyr::unnest(test_dirs)
```

So, this addition is correct, but please remove the now redundant entry from `Suggests`.